### PR TITLE
Add Individual-based modelling course to website

### LINKED ISSUE #1281: Individual-based modelling learning resources
@andrew-phillips-1  and Loveleeen have created a set of excellent course materials for learning individual-bases modelling:

https://docs.google.com/document/d/1fH0qKeLpvSRBNEePDYS9eaet4qJCQBOvr9kbJlyJVq4/edit#

@andrew-phillips-1  --- do you think we could present this content (with due attrribution of course) on our [webpag](https://www.tlomodel.org/background.html)...

### PR DESCRIPTION
**Pull request by tbhallett**
Fixes #1281

@andrew-phillips-1  - I've taken the [google doc ](https://docs.google.com/document/d/1fH0qKeLpvSRBNEePDYS9eaet4qJCQBOvr9kbJlyJVq4/edit#heading=h.2sm0fzjsv2wm)of the course and put in to .RST format in this PR>.

Some quick questions about this.....
* Please check the acknowledgment is correct. 
* Do you want to add a suggested citation?
* Should we remove what we have currently under item 0 on [this](https://www.tlomodel.org/background.html) page (i.e., your slides and video)  --- or should we put them at the top of the new page for this course? (Not sure of the relationship between the tutorial and the full course)

### CODE REVIEW

**Comment by andrew-phillips-1** [comment]
Please check the acknowledgment is correct.

I suggest you change to "This content was created by George Phillips, Loveleen Bansi-Matharu and Andrew Phillips"

Do you want to add a suggested citation?

How about "Phillips G, Bansi-Matharu, Phillips AN.  Building and Understanding Individual Based Models to Inform Health Policy.  https://github.com/UCL/TLOmodel/blob/hallett/add-ibm-course/docs/ibm_course.rst
 
Should we remove what we have currently under item 0 on [this](https://www.tlomodel.org/background.html) page (i.e., your slides and video) --- or should we put them at the top of the new page for this course? (Not sure of the relationship between the tutorial and the full course)

They are somewhat independent, so I suggest having them as two separate learning resources.

Thanks !

File: docs/learning.rst [hunk 1]
```diff
@@ -14,8 +19,6 @@ Health-Economics
 Modelling
 =========
 
-0. Tutorial introduction to individual-based modelled [`Video <https://youtu.be/Qwkcut56kro>`_] (:download:`Slides <./background/introduction-to-individual-based-models.pdf>`)
-
```
[the remark this answers is not in the document]
Comment (tbhallett) [reply docs/learning.rst:18]: Reverted that commit, thanks for clarifying

**Comment by tbhallett** [comment]
> This content was created by George Phillips, Loveleen Bansi-Matharu and Andrew Phillips

Thanks. I've added this. I didn't have a URL for George, so have left it as plain text.

**Comment by tbhallett** [comment]
Thanks @andrew-phillips-1  -- I think this is ready to go then.

**Review by tamuri — APPROVED** [review]

**Pull request merged**